### PR TITLE
Python example service doc

### PR DESCRIPTION
**Pull request by plauche**
* Describing how the example python payload service works
* Updating wording in example service (handler -> service)

### CODE REVIEW

File: docs/services/payload-service.rst [hunk 1]
```diff
@@ -1,4 +1,193 @@
-Payload Service
-===============
+***********************
+Example Payload Service
+***********************
 
-Ryan....please fill this out. 
\ No newline at end of file
+Payload Services are essentially Hardware Services but custom designed
```
Comment (jacoffey3) [inline docs/services/payload-service.rst]: caps...sorry I set a bad president

File: docs/services/payload-service.rst [hunk 2]
```diff
@@ -1,4 +1,193 @@
...
+            status
+        }
+    }
+
+Running the example
+===================
+
+Getting the example service up and running is fairly simple. First you must make sure you have the necessary python dependencies installed. If you are using the Kubos SDK vagrant box then these will already be installed. Otherwise you will need to run ``pip install -r requirements.txt``.
+
+Once the dependencies are in place you can run ``python service.py config.yml`` and the example service should begin. You will know that it is running if the command line output says ``* Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)``. You can now point a web browser to http://127.0.0.1:5000/graphiql to access a `graphical GraphQL interface <https://github.com/graphql/graphiql>`_. Here you can run quries and mutations against the GraphQL endpoints and see the results.
```
Comment (jacoffey3) [inline docs/services/payload-service.rst]: Add a section in the FAQs/Troubleshooting bits that talks about routing ports for the SDK.

File: docs/services/payload-service.rst [hunk 3]
```diff
@@ -1,4 +1,193 @@
-Payload Service
-===============
+***********************
+Example Payload Service
```
Comment (jacoffey3) [inline docs/services/payload-service.rst]: Remove Example. We might split it off later, but right now it's 1 doc.

File: docs/services/payload-service.rst (hunk 1 above)
Comment (jacoffey3) [reply docs/services/payload-service.rst]: Payload services.....hardware services...

File: docs/services/payload-service.rst [hunk 4]
```diff
@@ -1,4 +1,193 @@
...
+***********************
 
-Ryan....please fill this out. 
\ No newline at end of file
+Payload Services are essentially Hardware Services but custom designed
+for mission payload hardware. They share the same architecture as the hardware
+services exposing low level device APIs through a GraphQL interface.
```
Comment (CatGarab) [inline docs/services/payload-service.rst]: nit: comma after services
hyphenate low-level

**Comment by jacoffey3** [comment]
[Python and Rust](https://trello.com/c/tnEW6h1i/174-python-and-rust)

File: docs/services/payload-service.rst [hunk 5]
```diff
@@ -1,4 +1,193 @@
...
+
+``README.rst`` - Description file for service.
+
+``requirements.txt`` - Python module requirements file with list of module/version dependencies.
+
+``service.py`` - Boilerplate service code which reads the config file and starts up the GraphQL/HTTP endpoint.
+
+``service/`` - This folder holds the guts of the service's source.
+
+The service folder contains the main files which will need modifying when building a custom payload service.
```
Comment (CatGarab) [inline docs/services/payload-service.rst]: nit: change period to colon

File: docs/services/payload-service.rst [hunk 6]
```diff
@@ -1,4 +1,193 @@
...
+``schema.py`` - Contains the actual GraphQL models which are used to generate the GraphQL endpoint.
+
+
+We will now take a closer look at `models.py` and `schema.py` to see what exactly it takes to expose a hardware
+API through the service.
+
+models.py
+=========
+
+Inside of the example `models.py` file there is a `Subsystem` and `Status` class. Both of these classes must be subclasses of `graphql.ObjectType <http://docs.graphene-python.org/en/latest/types/objecttypes/>`_ from the `graphene <http://docs.graphene-python.org/en/latest/>`_ module.
```
Comment (CatGarab) [inline docs/services/payload-service.rst]: nit: -> "there are `Subsystem` and `Status` classes"

**Pull request merged**